app/main.ts

```diff
@@ -189,6 +215,16 @@ ws.onmessage = (signal: MessageEvent<string>) => {
 app.ticker.add(() => {
   //console.log("idPlayer: " + idPlayer + "\nturn: " + turn + "\nturnPhase: " + turnPhase + "\nisPlayerSpectator: " + isPlayerSpectator)
 
+  const winCondition = getWinCondition();
+  if (winCondition == WinCondition.PLAYER_0_WON) {
+    // Player 0 won
+    console.log("Player 1 won");
+  }
+  if ((winCondition == WinCondition.PLAYER_1_WON)) {
```

Comment:
if the player 1 won, the player 2 lost, so you can use `else if` toa void a second check
```suggestion
  else if ((winCondition == WinCondition.PLAYER_1_WON)) {
```